test/test_scenario_2.py

```diff
@@ -0,0 +1,72 @@
+# Unit and other tests
+from datetime import datetime
+import numpy as np
+import os
+import unittest
+import pandas as pd
+from shutil import copyfile
+
+from RAiDER.constants import Zenith
+from RAiDER.delay import tropo_delay
+from RAiDER.utilFcns import gdal_open, modelName2Module
+
+class RunTests(unittest.TestCase):
+
+    #########################################
+    # Scenario to use: 
+    # 1: GNSS station list
+    scenario = 'scenario_2'
+    weather_model_name = 'ERA5'
+    los = Zenith
+    download_only = False
+    verbose = True
+    wetName = 'stations_with_Delays.csv'
+
+    t = datetime(2020,1,3,23,0,0)
```

Comment:
Making these member variables is total overkill. Just pass the values in directly, using keyword arguments if you want to make it more readable. Generally providing test setup as members of a class like this is bad and leads to very messy tests. If you need to run some prior setup code, use a fixture (https://docs.pytest.org/en/stable/fixture.html), otherwise keep it inside of the test function itself.